article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -122,16 +122,45 @@ object ArticlePicker {
     isSupported
   }
 
-  def getTier(page: PageWithStoryPackage)(implicit request: RequestHeader): RenderType = {
+  def dcrShouldRender(request: RequestHeader): Boolean = {
+    // dcrShouldRender provides an override to let us force rendering by DCR even
+    // when an article is not supportted
+    val forceDCR = request.forceDCR
+    // our whitelist is only one article at the moment
+    val isInWhitelist = request.path == "/info/2019/dec/08/migrating-to-react"
+
+    forceDCR || isInWhitelist
```

Comment:
I'm sure the performance impact is minimal, but by calculating these both upfront we lose out on the short-circuit/lazy behaviour of `||`. I'd be alright with this being inlined to:

```scala
request.forceDCR || request.path == "/info/2019/dec/08/migrating-to-react"
```

I think it's still readable in that state.